test/e2e/specs/wp-gutenberg-experimental-features-spec.js

```diff
@@ -104,6 +104,25 @@ describe( `[${ host }] Experimental features we depend on are available (${ scre
 				);
 			}
 		);
+
+		// Regression test for https://github.com/Automattic/wp-calypso/pull/48940.
+		// At the time I write this, the default block patterns in the test site /
+		// theme used to test this (edge and non-edge) amount to 10. When activated,
+		// it goes up to 96. Testing if total is > 10 would be too brittle and too
+		// close to the default baseline number. That's why we actually test if the
```

Comment:
I guess that if we can say confidently that the number of default block patterns don't change often (does anyone know?), we can also change the check to fail if it sees the number is equal to the number of default block patterns thinking , which would also mean the ETK patterns haven't loaded.